docs/roadmap.md

```diff
@@ -0,0 +1,49 @@
+# Roadmap to Federated Learning on Azure ML
+
+## Contents
+Here we propose a roadmap for onboarding to Federated Learning (FL) on Azure ML. The roadmap follows the Crawl-Walk-Run<!-- -Fly--> framework, where we suggest to start with a simple example, and then progressively add more complexity.
+
+## I. :turtle: **Crawl** - Kick the tires and get a feel for the technology
+
+Your first few steps should be to provision a basic setup for FL and try to run a simple example. This will help you get a feel for the technology and the Azure ML platform. Great news, our [quickstart](./quickstart.md) will help you do just that!
```

Comment:
For each of the phases, let's formalize what is the starting point, who will be involved, and what outcome to expect.

Example here:

This will likely apply to any organization that hasn't done any Federated Learning training before, or that has practiced some FL using available open source frameworks in a simulated environment.

During this phase, we recommend you to get used to both the scientific and the infrastructure aspects of FL.

Do the following:

`<then insert a list of tutorials to try>`

Reply:
I followed this structure for the 3 phases. Please have a look